examples/all_widgets.rs

```diff
@@ -179,7 +177,7 @@ fn draw_ui(gl: &mut Gl,
             .rgba(0.4, 0.75, 0.6, 1.0)
             .frame(demo.frame_width)
             .label("PRESS")
-            .callback(Box::new(|| demo.bg_color = Color::random()))
+            .callback(box |&mut:| demo.bg_color = Color::random())
```

Comment:
I'm on `rustc 1.0.0-nightly (1d00c545e 2015-01-30 19:56:34 +0000)`, will update again now and see if it changes anything